assets/videos/resize.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+size=300
+mkdir -p ${size}px
+for file in $(ls ./original); do
+  echo $file
+  ffmpeg -i ./original/${file} -vcodec libx264 -vf "crop=min(iw\,ih):min(iw\,ih), scale=${size}:${size}" -an ./${size}px/${file%.*}.mp4
```

Comment:
Are you sure, this requires the video to be 1920x1080 as stated in the README? Looks to me like this will process videos of any size.

Reply:
You're right, this changed during development